package.json

```diff
@@ -30,6 +30,7 @@
     "bcrypt": "^5.1.1",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.0",
+    "csv": "^6.3.9",
```

Comment:
since you're only using the `stringfy` method you can use the `csv-stringify` package instead which is a subset of the csv package.

https://csv.js.org/stringify/
https://www.npmjs.com/package/csv